server/src/services/course-evaluation/course-evaluation.hooks.js

```diff
@@ -22,9 +24,9 @@ module.exports = {
         all: [addCoordinatorsAndReviewersToEvaluation()],
         find: [],
         get: [],
-        create: [addPermissionToUser()],
-        update: [],
-        patch: [],
+        create: [addPermissionToUser(), generateReport()],
+        update: [generateReport()],
```

Comment:
This works, but in the long-term we may want to change this so that reports are only generated when requested.